pac-aurora-provisioner/README.md

```diff
@@ -39,16 +43,24 @@ To provision a new PAC Aurora database cluster:
 
 - Generate credentials for the IAM user `pac-content-provisioner` in content-test aws account for a dev stack or in content-prod aws account for a staging/ prod stack.
 - Get the environment variables from the **pac-aurora-provisioner** LastPass note in the **Shared-PAC Credentials & Services Login Details** folder.
-- Set the `CLUSTER` environment variable, this will be appended to `pac-aurora` for all provisioned infrastructure. Note: The cluster name should be region agnostic, for example, `staging` will provision `pac-aurora-staging-eu` and `pac-aurora-staging-us` database instances.
+- Set the `CLUSTER` environment variable and the `CLUSTER_SUFFIX`, this will be appended to `pac-aurora` for all provisioned infrastructure. `CLUSTER_SUFFIX` will be appended to the environment name it is possible to provision a db without a suffix however for DR purposes you should always add a suffix (currently we are working off a list of composer name [here](https://docs.google.com/spreadsheets/d/1cyqrrC5T24EU3frwPtXe8xr3mrOPSNxxvSB88IDZsaE/edit#gid=0 "composer list")).
+  Note: The cluster name should be region agnostic, for example, `staging_xxx` will provision `pac-aurora-staging_xxx-eu` and `pac-aurora-staging_xxx-us` database instances.
```

Comment:
Unless there's a good reason, I think it would be easier if these _values_ have hyphens throughout (i.e. no underscores). The examples towards the bottom of this file use hyphens.